src/shared/services/IndexedDb.ts

```diff
@@ -4,15 +4,15 @@ import Utils from '../context/Utils';
 import Emitter from '../libraries/Emitter';
 import Log from '../libraries/Log';
 
-const DATABASE_VERSION = 5;
```

Comment:
maybe we should hold onto this and simply set it as the default in the constructor. but this pattern of setting the version of a constant will make it easier to find & update in the future

Reply:
ya keeping that as a constant make finding it easier. I added a `--fixup` commit to address it.